sites/public/src/pages/_error.tsx

```diff
@@ -20,24 +19,21 @@ const ErrorPage = () => {
   }, [profile])
 
   return (
-    <Layout>
```

Comment:
When this page was rendering (when the listing call fails on the listing view page) the header and footer were actually rendering twice.